### PR TITLE
Update copyright to 2018

### PR DESCRIPTION
**Pull request by chbatey**
I did spend a few minutes trying to use sbt-header but it failed in various ways and
didn't want ot burn time on it.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -218,7 +218,7 @@ For a pull request to be considered at all it has to meet these requirements:
 
 Some additional guidelines regarding source code are:
 
-- Files should start with a ``Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>`` copyright header.
+- Files should start with a ``Copyright (C) 2018 Lightbend Inc. <http://www.lightbend.com>`` copyright header.
```
Comment (raboof) [inline CONTRIBUTING.md]: I think we should switch to https for the URL

**Review by raboof — APPROVED** [review]
Change LGTM, perhaps update link to https (or postpone that  to another PR)

File: CONTRIBUTING.md (hunk 1 above)
Comment (chbatey) [reply CONTRIBUTING.md]: i'll update it

**Comment by chbatey** [comment]
Gah this conflicts with any moved/deleted file, someone else from @akka/akka-team approve it :)

**Review by johanandren — APPROVED** [review]
Epic!

**Pull request merged**